### PR TITLE
Fix  issues around "0" tickets in modal.

### PR DESCRIPTION
**Pull request by Camwyn**
🎫 https://central.tri.be/issues/129443#note-25

🎥 http://p.tri.be/BZTijy
🎥 http://p.tri.be/1XoutH

Requires https://github.com/moderntribe/tribe-common/pull/1182

### CODE REVIEW

File: src/resources/js/tickets-block.js [hunk 1]
```diff
@@ -204,9 +206,13 @@ tribe.tickets.block = {
...
 			quantity     = isNaN( quantity ) ? 0 : quantity;
-			footerAmount += obj.numberFormat( $price.text() ) * quantity;
+			var price = parseFloat( $price.text() );
+			console.log(price);
+			price = price * quantity;
+			console.log(price);
+			footerAmount += price;
 		} );
-
+		console.log(footerAmount);
```
Comment (sc0ttkclark) [inline src/resources/js/tickets-block.js]: Oops

File: src/resources/js/tickets-block.js [hunk 2]
```diff
@@ -204,9 +206,13 @@ tribe.tickets.block = {
 			var $price   = $( this ).closest( obj.selector.item ).find( obj.selector.itemPrice ).first();
 			var quantity = parseInt( $( this ).val(), 10 );
 			quantity     = isNaN( quantity ) ? 0 : quantity;
-			footerAmount += obj.numberFormat( $price.text() ) * quantity;
+			var price = parseFloat( $price.text() );
+			console.log(price);
+			price = price * quantity;
+			console.log(price);
```
Comment (sc0ttkclark) [inline src/resources/js/tickets-block.js]: Oops

File: src/resources/js/tickets-block.js [hunk 3]
```diff
@@ -204,9 +206,13 @@ tribe.tickets.block = {
 			var $price   = $( this ).closest( obj.selector.item ).find( obj.selector.itemPrice ).first();
 			var quantity = parseInt( $( this ).val(), 10 );
 			quantity     = isNaN( quantity ) ? 0 : quantity;
-			footerAmount += obj.numberFormat( $price.text() ) * quantity;
+			var price = parseFloat( $price.text() );
+			console.log(price);
```
Comment (sc0ttkclark) [inline src/resources/js/tickets-block.js]: Oops

Comment (sc0ttkclark) [reply src/resources/js/tickets-block.js]:
```suggestion
```

File: src/resources/js/tickets-block.js (hunk 2 above)
Comment (sc0ttkclark) [reply src/resources/js/tickets-block.js]:
```suggestion
```

File: src/resources/js/tickets-block.js (hunk 1 above)
Comment (sc0ttkclark) [reply src/resources/js/tickets-block.js]:
```suggestion
```

Comment (Camwyn) [reply src/resources/js/tickets-block.js]: doh!

**Pull request merged**